<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectMasterNoDialog</class>
 <widget class="QDialog" name="SelectMasterNoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Master number</string>
  </property>
  <widget class="QGroupBox" name="groupBoxMasterNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>マスター画像</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>マスター番号</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMasterNo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>21</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAlgorithm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>551</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>アルゴリズムからの画像</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>LibType</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DLLRoot</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DLLName</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LibName</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>キャンセル</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
